tests/unit/swagger/pathGeneration/optionsRoutes.spec.ts

```diff
@@ -0,0 +1,24 @@
+import 'mocha';
+import { MetadataGenerator } from '@tsoa/cli/metadataGeneration/metadataGenerator';
+import { SpecGenerator2 } from '@tsoa/cli/swagger/specGenerator2';
+import { getDefaultExtendedOptions } from 'fixtures/defaultOptions';
+import { VerifyPath } from 'unit/utilities/verifyPath';
+
+describe('OPTIONS route generation', () => {
```

Comment:
Can you add tests to verify that the created spec is correct here (Both OpenAPI 2 (SpecGenerator2) and OpenAPI 3 (SG3).

Reply:
@WoH I've notice that both [OpenAPI2](https://swagger.io/specification/v2/#path-item-object) and [3](https://swagger.io/specification/#path-item-object) requires options field in Path Item Object with value of Operation Object.  
So I think we only need to verify one version.  
Can you tell me where to put the `spec verify test`?, thx.  
I don't see other places to put such tests.

Reply:
https://github.com/lukeautry/tsoa/tree/master/tests/unit/swagger/pathGeneration

Reply:
@WoH I think I've already provide a tests in pathGeneration folder.